### DISCUSSION TITLE
Feature Request: Support passing arrays as parameters in Jdbc Api

### DISCUSSION THREAD

**Discussion by tolmalev** [duckdb/duckdb#11481] [Ideas]
For a query like `select from table where col IN (select * from unnest(values))` when list values is passed from the client we need a way to pass array as a parameter to Jdbc API.

Currently, method setArray in prepared statement throws an exception.

**Comment by StephenFlavin** [comment]
I was looking for this too, it's been implemented but not released yet see: https://github.com/duckdb/duckdb-java/pull/30
